services/database/serverless.yml

```diff
@@ -127,6 +129,33 @@ resources:
             AttributeType: S
           - AttributeName: md5Hash
             AttributeType: S
+    SarReportTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: ${self:custom.sarReportTableName}
+        PointInTimeRecoverySpecification:
+          PointInTimeRecoveryEnabled: true
+        StreamSpecification:
+          StreamViewType: NEW_AND_OLD_IMAGES
+        AttributeDefinitions:
+          - AttributeName: state
+            AttributeType: S
+          - AttributeName: id
+            AttributeType: S
+        KeySchema:
+          - AttributeName: state
+            KeyType: HASH
+          - AttributeName: id
+            KeyType: RANGE
+        BillingMode: PAY_PER_REQUEST # Set the capacity to auto-scale
+    SarFormBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: !Sub ${self:service}-${self:custom.stage}-sar
```

Comment:
might be safer if this referenced the `sarFormBucket` name from custom. Otherwise that variable isn't used!

Reply:
Good call on both of these! Great catch. 🙇